### DISCUSSION TITLE
invalid compression level

### DISCUSSION THREAD

**Discussion by Solaris-Chia** [madMAx43v3r/chia-gigahorse#348] [Q&A]
i try compression plotting.

Ryzen 9 5900x
128GB ram
2x NVME
NVIDIA GTX 1060

i always get the following message:
cuda_plot_k32.exe -n 2 -C 7 -S 2 -t F:\ -2 D:\ -d G:\ -M 64 -c xch1n57awf7pmg0dlgc5ppnmmua52y07xpvd4tccwxjqps7qvkhrqjuqnphqgp -f 86885d2825d92a739ae9bd9192f57a0f8d5c5e773ac543fef3acb5e225b5a9ecdaabb31aa19f14699cf66a3b7d8148f5 -W 1
Invalid compression level: 7

i tried Levels 1 to 9 and 11 to 20

i want to use 7 or 8 as i don't have a GPU on my farm

**Comment by Kruger1981** [comment]
Have you tried using the v3 of the cuda plotter?

**Reply by Solaris-Chia** [reply]
yes. just tried v3

E:\PLOTS\chia-gigahorse-master\cuda-plotter\windows>**cuda_plot_k32_v3.exe** -C 7 -n 1 -S 2 -t F:\ -2 D:\ -d G:\ -M 64 -c xch1n57awf7pmg0dlgc5ppnmmua52y07xpvd4tccwxjqps7qvkhrqjuqnphqgp -f 86885d2825d92a739ae9bd9192f57a0f8d5c5e773ac543fef3acb5e225b5a9ecdaabb31aa19f14699cf66a3b7d8148f5 -W 1
Invalid compression level: 7

**Reply by Solaris-Chia** [reply]
with GH 2.0 it works....

Chia k32 next-gen CUDA plotter - 3287c51
Plot Format: mmx-v2.5
Network Port: 8444 [chia]
No. GPUs: 1
No. Streams: 2
Direct IO: No
Final Destination: G:\
Bucket Chunk Size: 8 MiB
Max Pinned Memory: 58.8984 GiB
Number of Plots: 1
Initialization took 0.105 sec
Crafting plot 1 out of 1 (2024/12/15 13:53:42)
Process ID: 14076
Pool Puzzle Hash:  9d3dd727c1da1edfa3140867bdf3b4511fe3058daaf1871a400c3c065ae304b8
Farmer Public Key: 86885d2825d92a739ae9bd9192f57a0f8d5c5e773ac543fef3acb5e225b5a9ecdaabb31aa19f14699cf66a3b7d8148f5
Working Directory:   F:\
Working Directory 2: D:\
Compression Level: C7
Plot Name: plot-k32-c7-2024-12-15-13-53-7a86c6817b5c36169807aaf6a3161e553cb11f730009833579028b52b9113a8b
Created disk buffer D:\cuda_plot_tmp2_1734267222129912.tmp
[P1] Setup took 0.158 sec
[P1] Table 1 took 9.907 sec, 4294967296 entries, 16787589 max, 66678 tmp, 0 GB/s up, 3.331 GB/s down
[P1] Table 2 took 20.166 sec, 4295069580 entries, 16788785 max, 66534 tmp, 1.58683 GB/s up, 2.45464 GB/s down
[P1] Table 3 took 21.452 sec, 4295066626 entries, 16789283 max, 66639 tmp, 2.23761 GB/s up, 3.84581 GB/s down
